docs/content/en/docs/reference/artifacts.md

```diff
@@ -305,60 +305,57 @@ These steps use `image-builder` to create an Ubuntu-based or RHEL-based image fo
-    sudo tar xvf image-builder*.tar.gz
-    sudo cp image-builder /usr/local/bin
+    curl -#o- https://anywhere-assets.eks.amazonaws.com/releases/bundles/21/artifacts/image-builder/0.1.2/image-builder-linux-amd64.tar.gz | \
+      sudo tar -xzC /usr/local/bin ./image-builder
     ```
-1. Create a Bare Metal configuration file (for example, `baremetal.json`) to identify the location of a Red Hat Enterprise Linux 8 ISO image and related checksum and Red Hat subscription information:
+
+1. Create an Ubuntu or Red Hat image.
+
+   **Ubuntu**
+
+   Run `image-builder` with the following options:
+
+      * `--os`: `ubuntu`
+      * `--hypervisor`: For Bare Metal use `baremetal`
+      * `--release-channel`: Supported EKS Distro releases include 1-20, 1-21, 1-22, 1-23, and 1-24.
+
+      ```bash
+      image-builder build --os ubuntu --hypervisor baremetal --release-channel 1-24
+      ```
+
+   **Red Hat Enterprise Linux (RHEL)**
+
+   RHEL images require a configuration file to identify the location of the RHEL 8 ISO image and
+   Red Hat subscription information. The `image-builder` command will temporarily consume a Red
+   Hat subscritpion that is returned once the image is built.
+
    ```json
    {
      "iso_url": "<https://endpoint to RHEL ISO endpoint or path to file>",
      "iso_checksum": "<for example: ea5f349d492fed819e5086d351de47261c470fc794f7124805d176d69ddf1fcd>",
      "iso_checksum_type": "<for example: sha256>",
      "rhel_username": "<rhsm username>",
      "rhel_password": "<rhsm password>",
-     "extra_rpms": "<Space-separated list of RPM packages. Useful for adding required drivers or other packages>"
+     "extra_rpms": "<space-separated list of RPM packages; useful for adding required drivers or other packages>"
    }
    ```
-   >**_NOTE_**: To build the RHEL-based image, `image-builder` temporarily consumes a Red Hat subscription. That subscription is returned once the image is built.
-
-1. Create an ubuntu or redhat image:
-
-   * To create an Ubuntu-based image, run `image-builder` with the following options:
-
-      * `--os`: `ubuntu`
-      * `--hypervisor`: For Bare Metal use `baremetal`
-      * `--release-channel`: Supported EKS Distro releases include 1-20, 1-21, 1-22, 1-23, and 1-24.
-      * `--baremetal-config`: Bare Metal configuration file (`baremetal.json` in this example)
 
-      ```bash
-      image-builder build --os ubuntu --hypervisor baremetal --release-channel 1-24 --baremetal-config baremetal.json
-      ```
-   * To create a RHEL-based image, run `image-builder` with the following options:
+   Run the `image-builder` with the following options:
 
       * `--os`: `redhat`
-      * `--hypervisor`: For Bare Metal use `baremetal`
+      * `--hypervisor`: For Bare Metal use `baremetal`.
       * `--release-channel`: Supported EKS Distro releases include 1-20, 1-21, 1-22, 1-23, and 1-24.
```

Comment:
Same question about 1.20 still being supported.

Reply:
See other thread. Will hold this comment open so we don't forget.

Reply:
How about something like this: "...A supported EKS Distro release. For example, use 1-24 for Kubernetes release 1.24."

Reply:
Never mind. As we discussed, just use the same text from above.